feed.xml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Fair enough. Seemed like jekyll-feed had the same discussion earlier this year and it was an involved discussion, so I thought I might ask ;)

I'll try and get that implemented now.